public/_sidebar.md

```diff
@@ -10,3 +10,7 @@
 * [Pull Request Manual](/guides/PULL_REQUEST_GUIDE.md)
 * [Code of Conduct Guidelines](/guides/CODE_OF_CONDUCT_GUIDE.md)
 * [Security policy Guidelines](/guides/SECURITY_GUIDE.md)
+
+**SDK and Libraries**
+
+* [Introduction](/sdks/INTRODUCTION.md)
```

Comment:
Will/should the platform specific pages be listed here?

Reply:
Yes, but since it's just a template for the moment I haven't put anything on the sidebar. I'm also thinking about removing the link from the `Introduction` page before merging. It makes sense to add them only once they are ready.

Reply:
I've added them as a comment and commented out the `platform specific` section in the `Introduction` until at least an article is available.